fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/LoadCommand.java

```diff
@@ -0,0 +1,385 @@
+        this.comment = Objects.requireNonNull(comment, "comment should not null");
+    }
+
+    /**
+     * for test print
+     * @param ctx context
+     * @return parsed insert into plan
+     */
+    @VisibleForTesting
+    public List<LogicalPlan> parseToInsertIntoPlan(ConnectContext ctx) throws AnalysisException {
+        List<LogicalPlan> plans = new ArrayList<>();
+        for (BulkLoadDataDesc dataDesc : sourceInfos) {
+            ctx.getState().setNereids(true);
+            plans.add(completeQueryPlan(ctx, dataDesc));
+        }
+        return plans;
+    }
+
+    @Override
+    public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        this.profile = new Profile("Query", ctx.getSessionVariable().enableProfile);
+        // TODO: begin txn form multi insert sql
+        profile.getSummaryProfile().setQueryBeginTime();
+        for (BulkLoadDataDesc dataDesc : sourceInfos) {
+            ctx.getState().setNereids(true);
+            plans.add(new InsertIntoTableCommand(completeQueryPlan(ctx, dataDesc), Optional.of(labelName), false));
+        }
+        profile.getSummaryProfile().setQueryPlanFinishTime();
+        executeInsertStmtPlan(ctx, executor, plans);
+    }
+
+    private LogicalPlan completeQueryPlan(ConnectContext ctx, BulkLoadDataDesc dataDesc)
+            throws AnalysisException {
+        if (LOG.isDebugEnabled()) {
+            LOG.debug("nereids load stmt before conversion: {}", dataDesc.toSql());
+        }
+        // build source columns, add select and insert node.
+        List<String> sinkCols = new ArrayList<>();
+        // contains the map of olap column to FileFieldNames, ColumnMappingList, ColumnsFromPath.
+        Map<String, NamedExpression> originSinkToSourceMap = new LinkedHashMap<>();
+        OlapTable olapTable = getOlapTable(ctx, dataDesc);
+        buildSinkColumns(olapTable, dataDesc, sinkCols, originSinkToSourceMap);
+
+        Map<String, String> properties = getTvfProperties(dataDesc, bulkStorageDesc);
+        // mapping sink colum to mapping expression
+        if (!dataDesc.getColumnMappingList().isEmpty()) {
+            for (Expression mappingExpr : dataDesc.getColumnMappingList()) {
+                if (!(mappingExpr instanceof EqualTo)) {
+                    continue;
+                }
+                String mappingColumn = ((EqualTo) mappingExpr).left().toSql();
+                if (originSinkToSourceMap.containsKey(mappingColumn)) {
+                    Expression mappingExprAlias = ((EqualTo) mappingExpr).right();
+                    originSinkToSourceMap.put(mappingColumn, new UnboundAlias(mappingExprAlias, mappingColumn));
+                }
+            }
+        }
+
+        for (String columnFromPath : dataDesc.getColumnsFromPath()) {
+            sinkCols.add(columnFromPath);
+            // columnFromPath will be parsed by BE, put columns as placeholder.
```

Comment:
I think the columnFromPath should be parsed by FE, and after parsing, we only need to add the parsed constant value here

Reply:
> I think the columnFromPath should be parsed by FE, and after parsing, we only need to add the parsed constant value here

but now we pass the columnFromPath to BE  by 'path_partition_keys'
and the operator at BE will parse path value and compute with 'path_partition_keys'